apps/e2e/cypress/e2e/proposalAdministration.cy.ts

```diff
@@ -29,7 +30,13 @@ context('Proposal administration tests', () => {
   let createdInstrumentId: number;
 
   const instrument1 = {
-    name: faker.random.words(2),
+    name: `0000. ${faker.random.words(2)}`,
```

Comment:
Might be worth using `faker.string.alpha()` and `faker.string.alphanumeric()` instead? But I guess we may just swap all of the deprecated faker functions at once in the future.

Reply:
Yes I see that it is deprecated but we can do the swap at once in one single PR.